daemon/logger/fluentd/fluentd.go

```diff
@@ -24,10 +24,12 @@ type fluentd struct {
 }
 
 const (
-	name             = "fluentd"
-	defaultHostName  = "localhost"
-	defaultPort      = 24224
-	defaultTagPrefix = "docker"
+	name                             = "fluentd"
+	defaultHostName                  = "localhost"
+	defaultPort                      = 24224
+	defaultTagPrefix                 = "docker"
+	defaultIgnoreConnectErrorOnStart = false           // So that we do not break existing behaviour
+	defaultBufferLimit               = 8 * 1024 * 1024 // 8M buffer by default
```

Comment:
It doesn't seem like much, until you have 100+ containers all buffering 8MB due to an outage.
I think we should change it for our use-case here.